blog/2025-03-25-sentry-openfeature-integration.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: "sentry-openfeature-integration"
+title: "Sentry OpenFeature Integration"
+date: 2025-04-08
+authors: [jonathannorris]
+description: "Announcing Sentry's OpenFeature integration for enhanced feature flag observability"
+tags: [sentry, openfeature, "feature flags", observability]
+draft: false
+---
+
+We're excited to announce that Sentry has released OpenFeature Hooks for [JavaScript](https://docs.sentry.io/platforms/javascript/configuration/integrations/openfeature/) and [Python](https://docs.sentry.io/platforms/python/integrations/openfeature/), enabling developers to track feature flag evaluations directly in their error monitoring and performance tracking.
+
+## What is the Sentry OpenFeature Integration?
+
+The Sentry OpenFeature Hook automatically tracks feature flag evaluations in your application.
+When an error occurs, Sentry will include the state of all evaluated feature flags in the error report, providing crucial context for debugging and troubleshooting.
+
+This integration is currently in beta and only supports boolean flag evaluations.
```

Comment:
Do we need to do anything in Sentry to get this functionality? E.g. ask them to enable a feature flag? 🤭

Reply:
Which functionality? The hook is live, and anyone can use it now. They are just marking it as a beta feature.